http-watch.sh

```diff
@@ -27,32 +27,34 @@ function LOG_DATE {
 log_running_date=`date '+%Y-%m-%d %H:%M:%S'`
 }
 
+function DATE {
```

Comment:
I agree with you on this point heavy, I have made a change and replaced DATE with LOG_DATE so it's more exact on it's use during the logging phase of the script.

In regards to the date on the filename, I have gone ahead and updated this to become 'log_http-watch.txt' for now.

Reply:
> In regards to the date on the filename, I have gone ahead and updated this to become 'log_http-watch.txt' for now.

I know I may have mentioned _.txt_ at some point in the past as an example, but it would be my recommendation that we follow-suit with other similar log files with the /var/log/ directory and perhaps call it: **http-watch.log** instead.

This would follow the standard like other logs in /var/log/ :
- yum.log
- boot.log
- lfd.log

etc.

IMHO there's no need to have the date in the filename, each log entry will have a date/time stamp anyway and then log rotation (archiving of very old log entries) will be handled by an external log rotation process on the server anyway.